pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2778 @@
+/*
```

Comment:
More specificly, KubeRuntimeManager will consume this api by using the gRPC client in generated code.